src/components/Grid.js

```diff
@@ -5,9 +5,11 @@ import { StyledGrid } from "../styles/StyledGrid";
 
 function Grid({ grid }) {
   return (
-    <StyledGrid>
-      {grid.map((row) => row.map((cell, x) => <Square key={x} type={cell} />))}
-    </StyledGrid>
+    <div>
+      <StyledGrid>
+        {grid.map((row) => row.map((cell, x) => <Square key={x} type={cell} />))}
+      </StyledGrid>
+    </div>
```

Comment:
These divs aren't needed

```suggestion
    <StyledGrid>
      {grid.map((row) => row.map((cell, x) => <Square key={x} type={cell} />))}
    </StyledGrid>
```